code-challenges.test.js

```diff
@@ -23,24 +30,66 @@ const hitchhikersCharacters = [
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes in an array of objects
+const cappy = (element) => {
+  return element.map(value => {
+    // On this line, I want to use string interpertation to format the output as i see fit. We need to use 'dot' notation to single our the key of the information we're looking for. Then charAt() to capitalize our first character. Then add it all together 
+    return `${value.name.charAt(0).toUpperCase() + value.name.slice(1)} is ${value.occupation}.`;
```

Comment:
This will capitalize the first letter of the first word, but won't capitalize the second word.